addons/repo-proxy/common/src/main/java/org/commonjava/indy/repo/proxy/RepoProxyUtils.java

```diff
@@ -1,12 +1,22 @@
 /**
  * Copyright (C) 2013 Red Hat, Inc.
+ <<<<<<< HEAD
  * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  * <p>
  * http://www.apache.org/licenses/LICENSE-2.0
  * <p>
+ =======
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ >>>>>>> f3674f198... Controlling NPM metadata content by repo-proxy
```

Comment:
Conflict need to be resolved. :)

Reply:
Good catch. Did not get it as no error report because it is in comment.